CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # Node.js Agent for Google Cloud Trace ChangeLog
 
+## 2017-02-28, Version 1.0.0-rc1, @matthewloring
+
+### Notable changes
+
+This version introduces a new API for creating custom trace spans as well as an API for writing custom plugins
+to instrument modules so tracing information can be reported. Please check out our docs section for a full
```

Comment:
Ok I saw that github defaults to show 0.6 branch which only has images inside `docs` folder. The master one has the md files. I think it will still be handy to link to it from here though